cypress/support/commands.js

```diff
@@ -114,4 +114,12 @@ Cypress.Commands.add('setIntercepts', () => {
         fixture: 'getTextPreviewSuccessResponse'
     });
 
+    cy.intercept('GET', `/api/proxy/gov-notify/previewTemplate?templateType=${PRE_CALL_MESSAGE_TEMPLATE}`, {
+        fixture: 'getBulkTextPreviewSuccessResponse'
+    });
+
+    cy.intercept('POST', `/api/proxy/gov-notify/send-bulk-message`, {
+        fixture: 'sendBulkMessageResponse'
+    });
```

Comment:
I think you can have something like this instead of defining a fixture file with empty object:
```
cy.intercept('POST', `/api/proxy/gov-notify/send-bulk-message`, {
        statusCode: 200, 
        body:{}
     });
```

Reply:
cool, i like this!